### PR TITLE
CLC-5907 OEC: Omit timestamps in skipped-course logs, keep them out of the browser

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/CLC-5907

See issue comments for the motivation behind taking out some timestamps.

Also generally improve logging on the import task, and update the contextual help.

### CODE REVIEW

**Comment by paulkerschen** [comment]
Half a Travis.

File: app/models/oec/sis_import_task.rb [hunk 1]
```diff
@@ -10,31 +10,48 @@ def run_internal
...
       course_codes_by_ccn = {}
       cross_listed_ccns = Set.new
-      Oec::Queries.courses_for_codes(@term_code, course_codes, @opts[:import_all]).each do |course_row|
+
+      home_dept_rows = Oec::Queries.courses_for_codes(@term_code, course_codes, @opts[:import_all])
+      home_dept_rows_imported = 0
+      log :info, "SIS data returned #{home_dept_rows.count} course-instructor pairings under home department"
+      home_dept_rows.each do |course_row|
         if import_course(worksheet, course_row)
+          home_dept_rows_imported += 1
```
Comment (johncrossman) [inline app/models/oec/sis_import_task.rb]: I've just learned Ruby has no "++".

File: app/models/oec/task.rb [hunk 2]
```diff
@@ -156,10 +156,14 @@ def date_time_of_most_recent(category_name)
       nil
     end
 
-    def log(level, message)
+    def log(level, message, opts={})
       logger.send level, message
-      @log << "[#{Time.now.strftime '%T'}] #{message}"
-      write_status_to_cache if @opts[:log_to_cache]
+      if opts[:timestamp] == false
+        @log << "           #{message}"
```
Comment (johncrossman) [inline app/models/oec/task.rb]: Nice.

**Comment by johncrossman** [comment]
I approve more than half.

**Pull request merged**